.codeclimate.yml

```diff
@@ -0,0 +1,10 @@
+languages:
+  Ruby: true
+  JavaScript: true
+  PHP: true
+  Python: true
+exclude_paths:
+- "*.md"
+- "*.png"
+- "*.json"
```

Comment:
It was showing a bunch of files that didn't have any analysis associated with them. (Surprised Code Climate doesn't already filter these out by default.)